lib/src/includes/signed_video_auth.h

```diff
@@ -45,6 +45,8 @@ typedef enum {
   // validation.
   SV_AUTH_RESULT_OK = 4,
   // Successfully verified all NALUs that could be verified, and all expected NALUs are present.
+  SV_AUTH_RESULT_NO_PUBLIC_KEY = 5,
```

Comment:
I do not think this is needed. Better to use SV_AUTH_RESULT_SIGNATURE_PRESENT. See comment further down.